code/modules/research/xenobiology/crossbreeding/chilling.dm

```diff
@@ -0,0 +1,634 @@
+	if(!L.put_in_hands(gun))
+		qdel(gun)
+		user.visible_message("<span class='warning'>[src] flash-freezes [user]'s arm, cracking the flesh horribly!</span>")
+	else
+		user.visible_message("<span class='danger'>[src] chills and snaps off the front of the bone on [user]'s arm, leaving behind a strange, gun-like structure!</span>")
+	user.emote("scream")
+	L.apply_damage(30,BURN,which_hand)
+	..()
+
+/obj/item/slimecross/chilling/pink
+	colour = "pink"
+
+/obj/item/slimecross/chilling/pink/do_effect(mob/user)
+	user.visible_message("<span class='notice'>[src] cracks like an egg, and an adorable puppy comes tumbling out!</span>")
+	new /mob/living/simple_animal/pet/dog/corgi/puppy/slime(get_turf(user.loc))
+	..()
+
+/obj/item/slimecross/chilling/gold
+	colour = "gold"
+
+/obj/item/slimecross/chilling/gold/do_effect(mob/user)
+	user.visible_message("<span class='notice'>[src] lets off golden light as it melts and reforms into an egg-like device!</span>")
+	new /obj/item/capturedevice(get_turf(user.loc))
+	..()
+
+/obj/item/slimecross/chilling/oil
+	colour = "oil"
+
+/obj/item/slimecross/chilling/oil/do_effect(mob/user)
+	user.visible_message("<span class='danger'>[src] begins to shake with muted intensity!</span>")
+	addtimer(CALLBACK(src, .proc/boom), 50)
+
+/obj/item/slimecross/chilling/oil/proc/boom()
+	explosion(get_turf(src), -1, -1, 3, 10) //Large radius, but mostly light damage.
+	qdel(src)
+
+/obj/item/slimecross/chilling/black
+	colour = "black"
+
+/obj/item/slimecross/chilling/black/do_effect(mob/user)
+	if(ishuman(user))
+		user.visible_message("<span class='notice'>[src] crystallizes along [user]'s skin, turning into metallic scales!</span>")
+		var/mob/living/carbon/human/H = user
+		H.set_species(/datum/species/golem/random)
+	..()
+
+/obj/item/slimecross/chilling/lightpink
+	colour = "light pink"
+
+/obj/item/slimecross/chilling/lightpink/do_effect(mob/user)
+	user.visible_message("<span class='notice'>[src] blooms into a beautiful flower!</span>")
+	new /obj/item/clothing/head/peaceflower(get_turf(user.loc))
+	..()
+
+/obj/item/slimecross/chilling/adamantine
+	colour = "adamantine"
+
+/obj/item/slimecross/chilling/adamantine/do_effect(mob/user)
+	user.visible_message("<span class='notice'>[src] creaks and breaks as it shifts into a heavy set of armor!</span>")
+	new /obj/item/clothing/suit/armor/heavy/adamantine(get_turf(user.loc))
+	..()
+
+/obj/item/slimecross/chilling/rainbow
+	colour = "rainbow"
+
+/obj/item/slimecross/chilling/rainbow/do_effect(mob/user)
+	var/area/A = get_area(get_turf(user))
+	if(A.outdoors)
+		to_chat(user, "<span class='warning'>[src] can't effect such a large area.</span>")
+		return
+	user.visible_message("<span class='warning'>[src] reflects an array of dazzling colors and light, energy rushing to nearby doors!</span>")
+	for(var/obj/machinery/door/airlock/door in A)
+		new /obj/effect/forcefield/slimewall/rainbow(door.loc)
+	..()
+
+/////////////////////////////////////
+///////		MISC STUFF		/////////
```

Comment:
Split these into other files.

Reply:
Will do, especially since lots of other files have this kinda thing.